Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.cpp

```diff
@@ -214,18 +215,18 @@ void RemoteRenderingBackend::didCreateImageBuffer(Ref<ImageBuffer> imageBuffer)
     send(Messages::RemoteImageBufferProxy::DidCreateBackend(WTFMove(*handle)), imageBufferIdentifier);
 }
 
-void RemoteRenderingBackend::moveToSerializedBuffer(RenderingResourceIdentifier identifier)
+void RemoteRenderingBackend::moveToSerializedBuffer(RemoteSerializedImageBufferReference ref)
 {
     assertIsCurrent(workQueue());
     // Destroy the DisplayListRecorder which plays back to this image buffer.
-    m_remoteDisplayLists.take(identifier);
```

Comment:
So as an example: here it is undertandable that we destroy the remote image buffer by the identifier that identifies it.